### PR TITLE
Update content

### PR DESCRIPTION
**Pull request by SirSpidey**
### Description

Editorial review of content strings.

Make sure that these changes make sense in the stack context.

### Release required?

- [x] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

File: ibm_catalog.json [hunk 1]
```diff
@@ -26,30 +26,30 @@
 				"key protect",
 				"compliance"
 			],
-			"short_description": "This architecture supports deploying core security services - Key Protect, Secrets Manager, Security and Compliance Center. It will deploy Event Notifications and Observability.",
-			"long_description": "This architecture supports deploying core security services like Key Protect, Secrets Manager, Security and Compliance Center.",
+			"short_description": "Deploys core security services and some related services.",
+			"long_description": "This architecture deploys the following core security services: Key Protect, Secrets Manager, and Security and Compliance Center. It also deploys Event Notifications and Observability.",
```
Comment (ocofaigh) [inline ibm_catalog.json:30]: It also supports deploying the Security and Compliance Center Workload Protection instance

File: ibm_catalog.json [hunk 2]
```diff
@@ -26,30 +26,30 @@
...
 			"provider_name": "IBM",
 			"offering_icon_url": "https://globalcatalog.cloud.ibm.com/api/v1/1082e7d2-5e2f-0a11-a3bc-f88a8e1931fc/artifacts/terraform.svg",
 			"features": [
 				{
 					"title": "Creates an IBM Key Protect instance",
-					"description": "Creates and configures IBM Key Protect instance and creates root keys for Cloud Object Storage and Secrets Manager."
+					"description": "Creates and configures an IBM Key Protect instance and creates root keys for IBM Cloud Object Storage and Secrets Manager."
```
Comment (ocofaigh) [inline ibm_catalog.json:36]: root key also created for Event Notifications

File: ibm_catalog.json [hunk 3]
```diff
@@ -26,30 +26,30 @@
...
-					"title": "Creates an IBM Secrets Manager Instance ",
-					"description": "Creates and configures an IBM Secrets Manager Instance."
+					"title": "Creates an IBM Secrets Manager instance",
+					"description": "Creates and configures an IBM Secrets Manager instance."
 				},
 				{
-					"title": "Creates an IBM Security and Compliance Center Instance",
-					"description": "Creates and configures an IBM Security Compliance Center Instance."
+					"title": "Creates an IBM Security and Compliance Center instance",
+					"description": "Creates and configures an IBM Security Compliance Center instance."
```
Comment (ocofaigh) [inline ibm_catalog.json:44]: Maybe we need a new title for Security and Compliance Center Workload Protection instance ?

File: ibm_catalog.json [hunk 4]
```diff
@@ -26,30 +26,30 @@
...
-					"title": "Creates an IBM Cloud Object Storage Instance",
-					"description": "Creates and configures an IBM Cloud Object Storage Instance and multiple COS buckets that is encrypted by Key Protect"
+					"title": "Creates an IBM Cloud Object Storage instance",
+					"description": "Creates and configures an IBM Cloud Object Storage instance and multiple Object Storage buckets that is encrypted by Key Protect."
 				},
 				{
-					"title": "Creates Service to Service Authorizations.",
-					"description": "Creates and configures service-to-service authorizations for the following: KMS, COS, Secrets Manager, and SCC"
+					"title": "Creates service-to-service authorizations",
+					"description": "Creates and configures service-to-service authorizations for the following services: KMS, Object Storage, Secrets Manager, and Security and Compliance Center."
```
Comment (ocofaigh) [inline ibm_catalog.json:52]: Event Notifications too

File: ibm_catalog.json [hunk 5]
```diff
@@ -98,24 +98,24 @@
...
-								"title": "Creates IBM Cloud Core Security Services in a resource group.",
-								"description": "Creates a resource group called core-security-services or uses a preexisting one"
+								"title": "Creates IBM Cloud core security services in a resource group.",
+								"description": "Creates a resource group called `core-security-services` or uses an existing group."
 							},
 							{
-								"title": "Creates an IBM Key Protect Instance",
-								"description": "Creates and configures an IBM Key Protect Instance and creates root keys for Secret Manager and COS."
+								"title": "Creates an IBM Key Protect instance",
+								"description": "Creates and configures an IBM Key Protect instance and creates root keys for Secret Manager and IBM Cloud Object Storage."
```
Comment (ocofaigh) [inline ibm_catalog.json:106]: root key also created for Event Notifications

File: ibm_catalog.json [hunk 6]
```diff
@@ -98,24 +98,24 @@
...
-								"title": "Creates an IBM Secrets Manager Instance ",
-								"description": "Creates and configures an IBM Secrets Manager Instance"
+								"title": "Creates an IBM Secrets Manager instance ",
+								"description": "Creates and configures an IBM Secrets Manager instance."
 							},
 							{
-								"title": "Creates an IBM Security and Compliance Center Instance",
-								"description": "Creates and configures an IBM Security Compliance Center Instance"
+								"title": "Creates an IBM Security and Compliance Center instance",
+								"description": "Creates and configures an IBM Security Compliance Center instance."
```
Comment (ocofaigh) [inline ibm_catalog.json:114]: Maybe we need a new title for Security and Compliance Center Workload Protection instance ?

File: ibm_catalog.json [hunk 7]
```diff
@@ -212,7 +212,7 @@
 							"key": "sm_service_plan",
 							"type": "string",
 							"default_value": "standard",
-							"description": "The pricing plan to use when deploying the Secrets Manager Deployable Architecture",
+							"description": "The pricing plan to use for the Secrets Manager deployable architecture.",
```
Comment (ocofaigh) [inline ibm_catalog.json:215]: hmm, maybe don't say `deployable architecture` - some day it may come from module registry

**Review by ocofaigh — CHANGES_REQUESTED** [review]
Thanks @SirSpidey - left some comments

File: ibm_catalog.json (hunk 3 above)
Comment (SirSpidey) [reply ibm_catalog.json:44]: Added a feature for WP

File: ibm_catalog.json (hunk 6 above)
Comment (SirSpidey) [reply ibm_catalog.json:114]: Added a feature for WP

File: ibm_catalog.json (hunk 7 above)
Comment (SirSpidey) [reply ibm_catalog.json:215]: shortened just to `for Secrets Manager`, assuming that makes sense in this context.

**Comment by SirSpidey** [comment]
@ocofaigh updated based on your review. The feature descriptions aren't the most useful -- they really just add extra words to the title. But I have left them as is for the time being.

**Review by ocofaigh — APPROVED** [review]

**Pull request merged**